arm-recoveryservicesbackup/2016-12-01/swagger/backupManagement.json

```diff
@@ -43,7 +43,8 @@
         },
         "deprecated": false,
         "x-ms-pageable": {
-          "nextLinkName": "NextLink"
```

Comment:
Can you provide an example to validate this change? You can learn about `x-ms-examples` here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md.